model/template_designer/layout/GhostElement.js

```diff
@@ -0,0 +1,60 @@
+// Helpers
+const draggableLayerElementSelector = '.draggable-layer';
+const ghostElementSelector = '.ghost-element';
+
+export function addGhostElementToDraggableLayer({
+    top, left, width, height,
+}) {
+    const ghostElement = document.createElement('div');
+    const ghostElementStyle = `
```

Comment:
Can you leave top, left, height and width properties and set other ones in style file?